inc/classes/class_masterdelete.php

```diff
@@ -125,28 +158,40 @@ public function Delete($table, $idname, $id)
                 }
                 $func->question($q, $CurentURLBase. '&'. $idname .'='. $id, $_SESSION['md_referrer']);
             }
+
             return false;
         
         // Action
         } else {
             $res = $this->DoDelete($table, $idname, $id);
+
             if ($res) {
                 $func->confirmation(t('Der Eintrag wurde erfolgreich gelöscht'), $_SESSION['md_referrer']);
+
             } else {
                 $func->information(t('Der Eintrag konnte nicht gelöscht werden'), $_SESSION['md_referrer']);
             }
+
             unset($_SESSION['md_referrer']);
+
             return $res;
         }
     }
 
+    /**
+     *
+     * TODO Question for ASK_DELETE AND ASK_SET0
```

Comment:
there is AFAIK also `@ToDo`f for that

Reply:
Yes, many IDEs support both versions: `@todo` or `TODO`.
Afaik there is no strict convention over it.
I mostly search for "todo" which catches both.
I don't have a preference. For me it is just important, to mark this as a task.

Reply:
I could only find `@todo` the [PHPdoc documentation](http://docs.phpdoc.org/references/phpdoc/tags/todo.html), although `TODO` is mentioned for inline comments 🤷‍

Reply:
Should i change it or do you say "This is fine for now, next time use `@todo` in docblocks and `// TODO` in inline comments?

Reply:
I would suggest the second, given that this is a really minor point ;)